apps/dashboard/config/routes.rb

```diff
@@ -102,7 +102,7 @@
 
   get '/system-status', to: 'system_status#index', as: 'system_status' if Configuration.can_access_system_status?
 
-  post 'settings', :to => 'settings#update'
+  post 'settings/:action', to: 'settings#:action', as: 'settings'
```

Comment:
I can't quite discern why we need this change. Is it because of the id change I requested?

Reply:
This change is not related to your suggestions/changes.
This is an improvement I wanted to make to have specific methods in the controller to manage the updates to different types of settings.

The idea being that updating a profile will be managed by the `profile` method. Updating a announcement will be managed by the `announcement` method. This way we can apply logic specific to the type without having to add a lot of conditional logic to the previous single `update` method.

I can revert back to the previous way with the single method update a leave this for a later date.

Reply:
I have reverted the changes. The PR ready for a final review